src/bento/components/bento-autocomplete/1.0/test-e2e/test-bento-autocomplete.js

```diff
@@ -0,0 +1,79 @@
+describes.endtoend(
+  'bento-autocomplete',
+  {
+    version: '1.0',
+    fixture: 'bento/autocomplete.html',
+    environments: ['single'],
+  },
+  (env) => {
+    let controller;
+
+    beforeEach(() => {
+      controller = env.controller;
+    });
+
+    it('renders the autocomplete', async () => {
+      const element = await controller.findElement('bento-autocomplete');
+      await expect(controller.isElementDisplayed(element)).to.be.true;
+
+      const input = await controller.findElement('input#autocomplete');
+      await expect(controller.isElementDisplayed(input)).to.be.true;
+    });
+
+    describe('default autocomplete', () => {
+      it('displays options', async () => {
+        const element = await controller.findElement('bento-autocomplete');
+        const input = await controller.findElement('input#autocomplete');
+
+        controller.click(input);
+
+        await controller.switchToShadowRoot(element);
+
+        const results = await controller.findElement('[part="results"]');
+        await expect(controller.isElementDisplayed(results)).to.be.true;
+
+        const options = await controller.findElements('[role="option"]');
+        await expect(options.length).to.equal(3);
+      });
+
+      it('can select an option', async () => {
+        const element = await controller.findElement('bento-autocomplete');
+        const input = await controller.findElement('input#autocomplete');
+
+        controller.click(input);
+
+        await controller.switchToShadowRoot(element);
+
+        const option = await controller.findElement('[data-value="apple"]');
+
+        controller.click(option);
+
+        // await expect(controller.getElementAttribute(input, 'value')).to.equal(
+        //   'apple'
+        // );
```

Comment:
Is this meant to be part of the test?

Reply:
I'll remove it for now - I was trying to test the input value, but selenium doesn't seem to provide a way to do this for an input without an explicit `value` attribute. This specific case should be covered in the other tests anyway.